### PR TITLE
i18n(ko-KR): add `tutorial/1-setup/`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

add `tutorial/1-setup/`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/tutorial/1-setup/1.mdx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+<PreCheck>
+  - Astro 웹 사이트를 구축하는 데 사용할 도구를 설치하십시오.
+</PreCheck>
+
+## 필요한 개발 도구를 확보하세요
+
+### 터미널
+
+**명령줄 (터미널)**을 사용하여 Astro 프로젝트를 만들고 주요 명령을 실행하여 사이트를 빌드, 개발, 테스트하게 됩니다.
```
Comment (Yan-Thomas) [inline src/content/docs/ko/tutorial/1-setup/1.mdx:23]: Seems like the bold syntax failed here, could you give a look? I guess a space should do the trick.

image

**Review by Yan-Thomas — COMMENTED** [review]
Just a small nit, overall is LGTM.

File: src/content/docs/ko/tutorial/1-setup/1.mdx (hunk 1 above)
Comment (sarah11918) [reply src/content/docs/ko/tutorial/1-setup/1.mdx]: Fixed (I think). Good enough  :smile:

Comment (jsparkdev) [reply src/content/docs/ko/tutorial/1-setup/1.mdx]: There was a mistake. Thanks for fixing it!

**Pull request merged**